doc/content/gateways/dragino-lps8/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Dragino LPS8"
+description: ""
+weight: 
+---
+
+The **Dragino LPS8** is a LoRaWAN indoor gateway. Its operating system is based on the OpenWrt Linux system. The hardware includes an SX1308 LoRa concentrator that provides 10 programmable parallel demodulation paths. The gateway has an ethernet and WiFi backhaul. The Dragino LPS8 uses the Semtech legacy packet forwarder. The web-based management console allows you to configure and quickly connect the gateway with any LoRaWAN network. LPS8 supports EU868, IN865, US915, AU915, AS923, and KR920 ISM bands.
+
+This page guides you on how to connect the Dragino LPS8 to {{% tts %}}.
+
+![Dragino LPS8](dragino-lps8.png)
+
+## Prerequisites
+
+1. A user account on {{% tts %}} with user rights to add gateways.
+2. LPS8 connected to the internet and also your computer and the LPS8 gateway should be in the same network (E.g. your home WiFi network).
+3. The latest version of firmware running on the Dragino LPS8 (read the Dragino LPS8 [user manual](https://www.dragino.com/downloads/downloads/LoRa_Gateway/LPS8/LPS8_LoRaWAN_Gateway_User_Manual_v1.2.0.pdf) for instructions on firmware upgrade)
+
+## Gateway Settings on the LPS8
+
+Let’s get started with configuring the Dragino LPS8 LoRaWAN gateway using the web-based gateway configuration portal.
+
+1. Open a browser.
```

Comment:
We typically don't use numbered lists in our documentation.

Combine steps where appropriate, and use screenshots to break up steps. See https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/gateways/mikrotikrouterboard/_index.md for an example, and CONTRIBUTING.md for instructions to take a screenshot.

So this could be

```suggestion
Enter the IP address and port number of your gateway in the browser (the default port number is 8000, e.g `192.168.1.2.:8000`)
```